### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by mardet987**
Added the new forms as issue templates so that the issues can be submitted via github as an alternative to the mailing list.

### CODE REVIEW

**Comment by mardet987** [comment]
I'd like to suggest the forms as issue templates so that submissions could be made on github directly as an alternative to our mailing list (I wasn't aware of the issue templates feature until @mts299 mentioned it to me today!). Does anyone have any objections to this? If so, we can discuss at the next meeting.

**Comment by aburrell** [comment]
No, I think this is a great idea.

File: .github/ISSUE_TEMPLATE/dswg-issue-submission-form-template.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+---
```
Comment (aburrell) [inline .github/ISSUE_TEMPLATE/dswg-issue-submission-form-template.md]: Should this file be here?  I am having trouble spotting the differences between this one and the one below it.

**Review by aburrell — COMMENTED** [review]
A question about the first two files.

**Comment by mtwalach** [comment]
> I'd like to suggest the forms as issue templates so that submissions could be made on github directly as an alternative to our mailing list (I wasn't aware of the issue templates feature until @mts299 mentioned it to me today!). Does anyone have any objections to this? If so, we can discuss at the next meeting.

Great idea - I agree also!

File: .github/ISSUE_TEMPLATE/dswg-issue-submission-form-template.md (hunk 1 above)
Comment (mardet987) [reply .github/ISSUE_TEMPLATE/dswg-issue-submission-form-template.md]: you are correct - I deleted this file, thanks!

**Pull request merged**